### PR TITLE
EnableShiftPresence option no longer needed in -CsTeamsShiftsPolicy commands

### PR DESCRIPTION
**Pull request by martinicr**
The Shifts Service team is no longer supporting _shifts based presence_ in their PS commands. You can contact the following MSFT alias for futher details:
- wblocker
- martinflores
- erkandurmaz
- andreagannon

### CODE REVIEW

File: teams/teams-ps/teams/Get-CsTeamsShiftsPolicy.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Get-CsTeamsShiftsPolicy [[-Identity] <XdsIdentity>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-This cmdlet allows you to get properties of a TeamsShiftPolicy instance. Use this to get the policy name, user's shift based presence (EnableShiftPresence) and Teams off shift warning message-specific settings (ShiftNoticeMessageType, ShiftNoticeMessageCustom, ShiftNoticeFrequency, AccessGracePeriodMinutes).
+This cmdlet allows you to get properties of a TeamsShiftPolicy instance. Use this to get the policy name, and Teams off shift warning message-specific settings (ShiftNoticeMessageType, ShiftNoticeMessageCustom, ShiftNoticeFrequency, AccessGracePeriodMinutes).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsShiftsPolicy.md:22]:
```suggestion
This cmdlet allows you to get properties of a TeamsShiftPolicy instance. Use this to get the policy name and Teams off shift warning message-specific settings (ShiftNoticeMessageType, ShiftNoticeMessageCustom, ShiftNoticeFrequency, AccessGracePeriodMinutes).

```

File: teams/teams-ps/teams/Set-CsTeamsShiftsPolicy.md [hunk 2]
```diff
@@ -9,25 +9,25 @@ schema: 2.0.0
...
+Set-CsTeamsShiftsPolicy [[-Identity] <XdsIdentity>] [-ShiftNoticeFrequency <String>] [-ShiftNoticeMessageType <String>] [-ShiftNoticeMessageCustom <String>] [-AccessType <String>] [-AccessGracePeriodMinutes <Int64>] [-EnableScheduleOwnerPermissions <Boolean>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-This cmdlet allows you to set or update properties of a TeamsShiftPolicy instance. Use this to set the policy name, user's shift based presence (EnableShiftPresence) and Teams off shift warning message-specific settings (ShiftNoticeMessageType, ShiftNoticeMessageCustom, ShiftNoticeFrequency, AccessGracePeriodMinutes).
+This cmdlet allows you to set or update properties of a TeamsShiftPolicy instance. Use this to set the policy name, and Teams off shift warning message-specific settings (ShiftNoticeMessageType, ShiftNoticeMessageCustom, ShiftNoticeFrequency, AccessGracePeriodMinutes).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsShiftsPolicy.md:21]:
```suggestion
This cmdlet allows you to set or update properties of a TeamsShiftPolicy instance. Use this to set the policy name and Teams off shift warning message-specific settings (ShiftNoticeMessageType, ShiftNoticeMessageCustom, ShiftNoticeFrequency, AccessGracePeriodMinutes).

```

File: teams/teams-ps/teams/Set-CsTeamsShiftsPolicy.md [hunk 3]
```diff
@@ -9,25 +9,25 @@ schema: 2.0.0
...
 
 ## EXAMPLES
 
 ### Example 1
 ```powershell
-PS C:\> Set-CsTeamsShiftsPolicy -Identity OffShiftAccess_WarningMessage1_AlwaysShowMessage -EnableShiftPresence $true -ShiftNoticeMessageType Message1 -ShiftNoticeFrequency always -AccessGracePeriodMinutes 5
+PS C:\> Set-CsTeamsShiftsPolicy -Identity OffShiftAccess_WarningMessage1_AlwaysShowMessage -ShiftNoticeMessageType Message1 -ShiftNoticeFrequency always -AccessGracePeriodMinutes 5
 ```
-In this example, the policy instance is called "OffShiftAccess_WarningMessage1_AlwaysShowMessage", Shift based presence is enabled (On Shift, Off Shift), a warning message (Message 1) will be always be displayed to the user on opening Teams during off shift hours.
+In this example, the policy instance is called "OffShiftAccess_WarningMessage1_AlwaysShowMessage", a warning message (Message 1) will be always be displayed to the user on opening Teams during off shift hours.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsShiftsPolicy.md:30]:
```suggestion
In this example, the policy instance is called "OffShiftAccess_WarningMessage1_AlwaysShowMessage", a warning message (Message 1) will always be displayed to the user on opening Teams during off shift hours.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @martinicr, kindly commit the minor suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @get-itips @tseward

**Comment by dariomws** [comment]
@yogkumgit,

@get-itips and @tiburd are trying to contact the list of aliases for further details.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips Please update.

**Comment by get-itips** [comment]
@yogkumgit Still no feedback

**Comment by tiburd** [comment]
@get-itips @dariomws @tseward 
This PR is created by Martin Flores, @martinicr - if you need more information, please contact him.

Thanks!

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Pull request merged**